.config

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Default options
+readonly VALID_ROS_DISTROS="kinetic melodic noetic foxy dashing"
```

Comment:
Don't you think this should be in the `run-devel.sh` file. This should not be the user's responsibility.

Reply:
I prefer to have all constants in a single place, I few like this would make slightly easier making further changes. But I can move this to `run-devel.sh ` if desired.